From ffbce04c95525350a20c85fb80dcb0bc15f04de5 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Tue, 15 Mar 2022 12:03:17 +0000
Subject: [PATCH] GCC: make operators inline in AuctionRunner::InterestGroupKey

operators cannot be constexpr, because operators url::Origin are
not constexpr.
---
 content/browser/interest_group/auction_runner.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/content/browser/interest_group/auction_runner.h b/content/browser/interest_group/auction_runner.h
index 564a10c..5c52841 100644
--- a/content/browser/interest_group/auction_runner.h
+++ b/content/browser/interest_group/auction_runner.h
@@ -40,10 +40,10 @@ class CONTENT_EXPORT AuctionRunner {
   // appropriate.
   struct InterestGroupKey {
     InterestGroupKey(url::Origin o, std::string n) : owner(o), name(n) {}
-    constexpr bool operator<(const InterestGroupKey& other) const {
+    inline bool operator<(const InterestGroupKey& other) const {
       return owner != other.owner ? owner < other.owner : name < other.name;
     }
-    constexpr bool operator==(const InterestGroupKey& other) const {
+    inline bool operator==(const InterestGroupKey& other) const {
       return owner == other.owner && name == other.name;
     }
     url::Origin owner;
-- 
2.34.1

